src/clientiface.cpp

```diff
@@ -180,7 +180,7 @@ void RemoteClient::GetNextBlocks (
 	if (camera_fov <= 0) camera_fov = (72.0*M_PI/180) * 4./3.;
 
 	const s16 full_d_max = MYMIN(g_settings->getS16("max_block_send_distance"), wanted_range);
-	const s16 d_opt = MYMIN(g_settings->getS16("block_send_optimize_distance"), wanted_range);
+	const s16 d_opt = g_settings->getS16("block_send_optimization");
```

Comment:
This is no longer a distance, it's just a number, so -> `s16 opt_val` or similar.